docs/usage.rst

```diff
@@ -395,6 +395,15 @@ This will skip checking the hash of ``content`` and ``content_type`` only if
 the ``Authorization`` header omits the ``hash`` attribute. If the ``hash``
 attribute is present, it will be checked as normal.
 
+For requests whose ``content`` (and by extension ``content_type``) is ``None``
```

Comment:
This is great, thanks. I expanded on it a bit to add more context. Can you add this?

```diff
diff --git a/docs/usage.rst b/docs/usage.rst
index 4704d2c..16a7339 100644
--- a/docs/usage.rst
+++ b/docs/usage.rst
@@ -395,10 +395,17 @@ This will skip checking the hash of ``content`` and ``content_type`` only if
 the ``Authorization`` header omits the ``hash`` attribute. If the ``hash``
 attribute is present, it will be checked as normal.
 
+Empty requests
+==============
+
 For requests whose ``content`` (and by extension ``content_type``) is ``None``
 or ``''``, it is acceptable for the sender to omit the declared hash,
 regardless of the ``accept_untrusted_content`` value provided to the
-:class:`mohawk.Receiver`. If the ``hash`` attribute is present and
+:class:`mohawk.Receiver`.
+For example, a ``GET`` request typically has empty content and some
+libraries may or may not hash the content.
+
+If the ``hash`` attribute *is* present for an empty request and
 ``accept_untrusted_content`` is ``False``, a ``None`` value for either
 ``content`` or ``content_type`` will be coerced to ``''`` prior to hashing.
 This is to account for some dependent libraries that may provide the empty
```